lib/normalize_country.rb

```diff
@@ -36,12 +36,42 @@ def to_h(key, options = {})
       hash
     end
 
+    def extend_countries(*args)
+      args = args.flatten
+      return if args.empty?
+
+      if args.first.is_a?(Hash)
+        args.each { |mapping| add_country(mapping) }
+      else
+        load_from_yaml(args.first)
+      end
+    end
+
+    def reset!
+      Countries.clear
+      remove_instance_variable(:@to) if defined?(@to)
+      remove_instance_variable(:@formats) if defined?(@formats)
+
```

Comment:
I wrote just so, because in some cases expression on the right side of the operator `||=` will be executed again until value is `nil`.
A'm agree that `@to = @formats = nil` will also works fine in this case.